components/match2/wikis/brawlstars/match_group_input_custom.lua

```diff
@@ -338,10 +352,16 @@ function mapFunctions.getParticipantsData(map)
 
 	local maximumPickIndex = 0
 	for opponentIndex = 1, MAX_NUM_OPPONENTS do
-		for pickKey, brawler in Table.iter.pairsByPrefix(map, 't' .. opponentIndex .. 'p') do
+		for playerKey, player in Table.iter.pairsByPrefix(map, 't' .. opponentIndex .. 'p') do
+			local playerIndex = tonumber(string.sub(playerKey, -1))
```

Comment:
Do you want pairsByPrefix to return the index as well? Quite easy to add it as a 3rd output.

Reply:
hmm
idk i think it is not necessary that often
on the other hand i think it wouldn't hurt

Reply:
I've Opened a PR for it. Still need to test it however. 
Either wait it to be merged, or add a comment that it's only up to 9 players that are supported.